packages/@aws-cdk/aws-kinesisfirehose-alpha/test/integ.delivery-stream.source-stream.ts

```diff
@@ -6,13 +6,16 @@ import * as cdk from 'aws-cdk-lib';
 import * as constructs from 'constructs';
 import * as firehose from '../lib';
 import * as source from '../lib/source';
+import { AwsApiCall, ExpectedResult, IntegTest } from '@aws-cdk/integ-tests-alpha';
 
 const app = new cdk.App();
 
-const stack = new cdk.Stack(app, 'aws-cdk-firehose-delivery-stream-source-stream');
+const stack = new cdk.Stack(app, 'aws-cdk-firehose-delivery-stream-source-stream', { env: { region: 'us-east-1' } });
 
 const bucket = new s3.Bucket(stack, 'Bucket', {
+  bucketName: 'firehose-delivery-stream-source-stream-bucket',
```

Comment:
This is because I was seeing this error:
```
Error: Resolution error: Resolution error: Resolution error: Cannot use resource 'aws-cdk-firehose-delivery-stream-s3-all-properties/Bucket' in a cross-environment fashion, the resource's physical name must be explicit set or use PhysicalName.GENERATE_IF_NEEDED.
```
but setting `PhysicalName.GENERATE_IF_NEEDED` for the resource names was not working. I can try to test it again though because I may have been dealing with multiple issues at the time when I was seeing that error.

Reply:
Ok tried replacing it with `PhysicalName.GENERATE_IF_NEEDED` and got the same error as before.

Reply:
Actually I managed to change it. It seems like it was because I had set the region specifically that it was requiring me to provide a physical name. I think the region thing was some other issue but it's no longer happening so I can remove both.

Reply:
Thank you for looking further into the problem!